__tests__/api.test.js

```diff
@@ -0,0 +1,25 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /topics", () => {
+  test("returns status 200 and JSON object with key of topics and array of all the topics with description and seed", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .expect("Content-Type", "application/json; charset=utf-8")
+      .then((res) => {
+        res.body.topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+        expect(res.body.hasOwnProperty("topics")).toBe(true);
```

Comment:
We wouldn't need this assertion as we will only get to this line if the "topics" property exists on the response body. It has to exist in order for the assertions on lines 19 and 20 to work.